tests/contest/runtimetest/src/main.rs

```diff
@@ -33,6 +33,7 @@ fn main() {
         "hello_world" => tests::hello_world(&spec),
         ////////// ANCHOR_END: example_runtimetest_main
         "readonly_paths" => tests::validate_readonly_paths(&spec),
+        "masked_paths" => tests::validate_masked_paths(&spec),
```

Comment:
BTW, I realized `validate_masked_paths` should always be called, even if the subcommand isn't passed.

Reply:
It does seem necessary to call it every time. As for how to fix it, I believe the approach would be as follows, but I'm not entirely sure about the correct way to handle it, so I have a question.  

The idea is to remove the processing from the `subcommand` call and instead add an implementation that explicitly calls `subcommand`. However, this would likely result in the removal of the `masked_paths` test group. I'm unclear on where the test cases that were originally part of the `masked_paths` test group should be moved.

Reply:
You don't need to take care of it in this PR ;)

Reply:
If you are interested in it, please do it in another PR.